core/src/main/java/org/frankframework/http/HttpSender.java

```diff
@@ -474,6 +308,7 @@ public void setFirstBodyPartName(String firstBodyPartName) {
 	 * <ul>
 	 * <li>name: optional, used as 'filename' in Content-Disposition</li>
 	 * <li>sessionKey: mandatory, refers to contents of part</li>
+	 * <li>value: optional, the contents of the part if the sessionKey specified contains no data</li>
```

Comment:
Eigenlijk vind ik dat je dan value in parameter moet vullen, dat is namelijk daar automatisch al de default 😞

Reply:
Tsja, ik heb dit toegevoegd omdat dat is hoe het werkt in de (deprecated) MultiPartHttpSender. Nu werkt de HttpSender helemaal hetzelfde als de MultiPartHttpSender, dus is het compatibel.

Reply:
Ja maar dat ding is met een reden gedeprecate (sinds de 7.8), en nu zit deze funcitonaliteit op meerdere plekken in het framework.